sqlx-macros/src/query/mod.rs

```diff
@@ -271,13 +271,16 @@ where
         record_tokens
     };
 
-    let ret_tokens = quote! {{
-        use sqlx::Arguments as _;
+    let ret_tokens = quote! {
+        #[allow(clippy::all)]
+        {
+            use sqlx::Arguments as _;
 
-        #args_tokens
+            #args_tokens
 
-        #output
-    }};
+            #output
+        }
+    };
```

Comment:
Might want to wrap the content in an extra pair of brackets or else there might be stability errors about attributes on expressions again.

Reply:
sure thing!

Reply:
@o0Ignition0o sorry I meant an extra pair of brackets around the `#[allow(clippy::all)]`:

```rust
{
    #[allow(clippy:all)]
    {
        ...
     }
}
```

Reply:
Oh sorry my bad ^^'